### ISSUE TITLE
wskadmin cli returns 'Exception:  object has no attribute "read"'

### ISSUE THREAD

**Issue by mdeuser** [apache/incubator-openwhisk#2016]
## Environment details:

* Found on native ubuntu during build tests running a `wskadmin whois` command

## Steps to reproduce the issue:

1.   Failure is intermittent and somewhat rare.  A request() failure during `wskadmin whois` command will cause the error handling exception.  May be able to hard code a request failure to force the exception.

## Provide the expected results and outputs:

```
Non-zero exit code with meaningful message about the request failure.
```

## Provide the actual results and outputs:

```
    Exception:  object has no attribute "read"
    Informative: ....
```

**Comment by mdeuser** [comment]
Should also modify wsdamin code to dump out the root exception to see what the real root cause failure is.

wskutil.py
```
def request(...)
...
    except Exception as e:
        res = dict2obj({ 'status' : 500, 'error': str(e) })
        return res
```

**Comment by rabbah** [comment]
@mdeuser is this still an issue?

**Comment by mdeuser** [comment]
@rabbah - i've not run this command in a while.  were you able to recreate the issue or possibly make an educated guess in the code as to how the above error output might appear?

**Comment by rabbah** [comment]
I'm guessing fixed - I use wskadmin regularly and can use whois. There is one outstanding issue to be addressed by #2140. I'm going to close then.
